File: pset6/mario_more.py
import cs50

def main():
    while True:
        height=cs50.get_int("Height: ")
        if height>=0 and height<=23:
            break
    for i in range(1,height+1):
        print(" "*(height-i),end="")
        print("#"*i,end="")
        print(" "*2,end="")
        print("#"*i)

if __name__=="__main__":
    main()

File: pset1/mario_more.c
#include<stdio.h>
#include<cs50.h>

int main(){
    int x;
    do{
        printf("Height: ");
        x=get_int();
    }while(x<0||x>23);
    for(int i=1;i<=x;i++){
        for(int j=0;j<x-i;j++) printf(" ");
        for(int j=0;j<i;j++) printf("#");
        printf("  ");
        for(int j=0;j<i;j++) printf("#");
        printf("\n");
    }
}